GVFS/GVFS.Service/RepoRegistry.cs

```diff
@@ -21,12 +21,21 @@ public class RepoRegistry
         private ITracer tracer;
         private PhysicalFileSystem fileSystem;
         private object repoLock = new object();
-
-        public RepoRegistry(ITracer tracer, PhysicalFileSystem fileSystem, string serviceDataLocation)
+        private IRepoMounter mountProcess;
+        private GVFSPlatform gvfsPlatform;
+
+        public RepoRegistry(
+            ITracer tracer,
+            PhysicalFileSystem fileSystem,
+            string serviceDataLocation,
+            GVFSPlatform gvfsPlatform,
+            IRepoMounter mountProcess)

```

Comment:
This goes back to my naming nit earlier. It's awkward to have an interface type with a variable name referencing the concrete type.

Reply:
Renamed it to `repoMounter`